pages/admin/challenge/select-mode.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { components } from '../../../components/challenge';
+import Layout from '../../../components/layout';
+
+const { ModeSelect } = components;
+
+const SelectModePage = () => (
+    <Layout selectedMenuItem="challenge">
+        <ModeSelect next='/admin/challenge/choose-opponent' />
+    </Layout>
+);
+
```

Comment:
strings in ` constants.js`

Reply:
can't exactly use constants here, because this is the route page, not a component